app/services/applications/accept.rb

```diff
@@ -127,11 +129,9 @@ def fallback_schedule
     def validate_permitted_schedule_for_course
       return if errors.any?
       return if schedule_identifier.blank?
-      return unless schedule
+      return if schedule&.course_group&.courses&.include?(course)
```

Comment:
Can we accept an application not belonging to a `CourseGroup`? Is it also possible to have a `schedule` that is `nil` at this point? If it is not possible, then it is probably better not to protect ourselves against `nil` values because we would be making the investigation of issues a bit more difficult.

Reply:
Well said @padv2010 .. I've just rolled back this change 👍 😄  tks